packages/shared-component--breadcrumbs/src/breadcrumbs.pcss

```diff
@@ -10,19 +10,13 @@
   }
 }
 
-.coop-c-breadcrumbs__list-item {
-  position: relative;
-  padding-right: var(--spacing-32);
-}
-
 .coop-c-breadcrumbs__link {
   display: inline-block;
   padding: var(--spacing-4) 0;
   white-space: normal;
 }
 
-.coop-c-breadcrumbs__link + .coop-c-icon {
-  position: absolute;
-  top: var(--spacing-4);
-  right: 0;
-}
+.coop-c-breadcrumbs__arrow {
+  display: inline-block;
+  transform: translateY(.4rem);
```

Comment:
Can we solve this with margins, or even `position: relative` and left/right values?

I feel like "translate" makes a new compositing layer, and you often get a softer blur on it?